conans/client/hook_manager.py

```diff
@@ -4,6 +4,8 @@
 from conan.errors import ConanException
 
 valid_hook_methods = ["pre_export", "post_export",
+                      "pre_validate_build", "post_validate_build",
+                      "pre_validate", "post_validate",
```

Comment:
If we can cover the real-world use cases with just 1 of these hooks, I'd go for that, do the minimal changes to satisfy the need.

Reply:
So far, we have `pre_validate` requested in CCI as a real case, so the CI could skip the build for those cases when the glibc version is too old but the project requires a newer version (e.g. Qt), without injecting changes in the validate() which is really consumed by users. 

For other cases, @jcar87 can tell us more about it.

Reply:
@memsharded I'll drop the hook for `validate_build`.

Reply:
Done, removed validate_build hook on the commit a71eaaa8b